caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -18,6 +18,8 @@
 from builtins import range
 from builtins import str
 from builtins import zip
+#from numba import njit
```

Comment:
I'm removing this since we're not using it yet (and it's not part of caiman's requirements).